### PR TITLE
Model generator doesn't respect Ruby file naming

### PR DESCRIPTION
**Pull request by nguyenngoc2505**
- Use underscored model file name
Base on https://github.com/lotus/lotus/issues/225

### CODE REVIEW

**Comment by hieuk09** [comment]
:+1:

File: lib/lotus/generators/model/entity.rb.tt [hunk 1]
```diff
@@ -1,5 +1,3 @@
-require 'lotus/entity'
```
Comment (AlfonsoUceda) [inline lib/lotus/generators/model/entity.rb.tt]: @nguyenngoc2505 why did you remove those lines?

**Comment by AlfonsoUceda** [comment]
@nguyenngoc2505 you have to test the new behaviour. You have to assert new files don't generated like  `spec/bookshelf/repositories/AncientBook_repository_spec.rb` only this way `spec/bookshelf/repositories/ancient_book_repository_spec.rb`

**Comment by nguyenngoc2505** [comment]
@AlfonsoUceda Yes, I've already tested this. It will return `spec/bookshelf/repositories/ancient_book_repository_spec.rb`

File: lib/lotus/generators/model/entity.rb.tt (hunk 1 above)
Comment (nguyenngoc2505) [inline lib/lotus/generators/model/entity.rb.tt]: @AlfonsoUceda , @jodosha has written in here(https://github.com/lotus/lotus/issues/225#issuecomment-104054426) to remove this line.

**Comment by AlfonsoUceda** [comment]
Sorry but I don't see the test, can you paste the line, please? Thanks ;)

**Comment by nguyenngoc2505** [comment]
Sorry, I misunderstood your mean. I'll write it.

File: test/integration/cli/generate_test.rb [hunk 2]
```diff
@@ -81,12 +81,24 @@ def after
...
+        end
+      end
 
-      it 'generates model' do
-        @root.join('lib/delivery/entities/test.rb').must_be                      :exist?
-        @root.join('lib/delivery/repositories/test_repository.rb').must_be       :exist?
-        @root.join('spec/delivery/entities/test_spec.rb').must_be                :exist?
-        @root.join('spec/delivery/repositories/test_repository_spec.rb').must_be :exist?
+      describe 'when model names are CamelCase names.' do
+        let(:klass) { 'Test' }
```
Comment (AlfonsoUceda) [inline test/integration/cli/generate_test.rb]: Instead `Test` can be `TestCase` because with this test you assert correct camelcase an underscore

Comment (jodosha) [inline test/integration/cli/generate_test.rb]: @nguyenngoc2505 I agree with @AlfonsoUceda :+1:

**Comment by nguyenngoc2505** [comment]
@jodosha @AlfonsoUceda Thanks!. I've updated. Please review this.

**Comment by AlfonsoUceda** [comment]
:+1:

**Comment by jodosha** [comment]
@nguyenngoc2505 Just merged this, thank you very much! :+1:
